src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from 'react';
+import { getPeople } from '../api';
+import { PeopleList } from './PeopleList';
+
+type Person = {
+  name: string;
+  sex: string;
+  born: number;
+  died: number;
+  mother: string;
+  father: string;
+};
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [error, setError] = useState<string>('');
+  const [loading, setLoading] = useState(false);
+
+  useEffect(() => {
+    setLoading(true);
+    getPeople()
+      .then(fetchedPeople => {
+        setLoading(false);
+
+        setPeople(fetchedPeople);
+      })
+      .catch(() => {
+        setLoading(false);
+
+        setError('Something went wrong');
+      });
+  }, []);
+
+  if (error === '') {
+    return <PeopleList people={people} isLoading={loading} />;
+  } else if (error !== '') {
```

Comment:
The condition `if (error === '')` and `else if (error !== '')` can be simplified. Since these are complementary conditions, you can use `if (error)` for the error case and `else` for the non-error case, which makes the code cleaner and more readable.